yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -31,6 +31,9 @@ export class Sequencer {
   private pollingIntervalMs: number = 1000;
   private maxTxsPerBlock = 32;
   private minTxsPerBLock = 1;
+  // TODO: zero values should not be allowed for the following 2 values in PROD
```

Comment:
Similar to above. Should we look at constraining it to not be 0?

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/4421#discussion_r1481101311